extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -16,153 +16,95 @@
 
 import {AmpAd3PImpl} from '../amp-ad-3p-impl';
 import {createIframePromise} from '../../../../testing/iframe';
+import {stubService} from '../../../../testing/test-helper';
 import {createElementWithAttributes} from '../../../../src/dom';
-import {markElementScheduledForTesting} from '../../../../src/custom-element';
 import * as adCid from '../../../../src/ad-cid';
 import '../../../amp-sticky-ad/0.1/amp-sticky-ad';
 import * as sinon from 'sinon';
-import * as lolex from 'lolex';
 
+function createAmpAd(win) {
+  const ampAdElement = createElementWithAttributes(win.document, 'amp-ad', {
+    type: '_ping_',
+    width: 300,
+    height: 250,
+    src: 'https://testsrc',
+    'data-valid': 'true',
+    'data-width': '6666',
+  });
+  ampAdElement.getIntersectionChangeEntry = () => {};
+  ampAdElement.getPlaceholder = () => {};
+  ampAdElement.getFallback = () => {};
+  return new AmpAd3PImpl(ampAdElement);
+}
 
 describe('amp-ad-3p-impl', () => {
   let sandbox;
+  let ad3p;
+  let win;
 
   beforeEach(() => {
     sandbox = sinon.sandbox.create();
+    return createIframePromise().then(iframe => {
```

Comment:
I don't think this is valid anymore. We should just have separate test cases for w/ and w/o CID